setup.py

```diff
@@ -221,6 +226,8 @@ def run(self):
             if os.path.isfile(name):
                 os.remove(name)
 
+        shutil.rmtree(dir_latex_antlr, ignore_errors=True)
```

Comment:
I don't like this. If you run `setup.py clean` with this it will delete checked in files.